src/main/resources/db/changesets/20200625_initial_schema.yaml

```diff
@@ -50,18 +50,25 @@ databaseChangeLog:
               name: wait_until
               type: timestamp
               constraints:
-                nullable: false
+                nullable: true
                 unique: false
```

Comment:
I think we can remove this column as Stairway will handle waiting during a flight.